onnx/defs/math/defs.cc

```diff
@@ -549,3 +549,50 @@ Given two equivalent values, this operator uses the indices along the axis  as
       " axis",
       AttributeProto::INT,
       static_cast<int64_t>(-1));
+
+
+ONNX_OPERATOR_SCHEMA(Unique)
+  .SetDoc(R"DOC(
+Returns the unique scalar elements of the input tensor as a 1-D tensor. The
+order of the elements in the output tensor is unspecified.
+)DOC")
+  .Input(0, "X", "Tensor to find unique elements of", "T")
+  .Output(
+    0,
+    "Y",
+    "1-D tensor containing unique elements of the input tensor.",
+    "T")
+  .Output(
+    1,
+    "inverse_indices",
```

Comment:
The name of `inverse_indices` look weird, from the outside it tell me that it inverses the result and not returning the indices as an extra output. Shouldn't we call it `return_indices`?

Reply:
sure